.typos.toml

```diff
@@ -11,6 +11,7 @@ ue = "ue" # elastic stretch tensor
 ser = "ser" # "switched evolution relaxation" (used for pseudo transient continuation)
 ded = "ded" # derivative w.r.t. electrode variables
 uncomplete = "uncomplete" # used in the context of the uncomplete() call fo a sparse matrix
+Equil = "Equil" # "Equilibration" (used to pre-order a linear system)
```

Comment:
Can't we just use the full word instead of our own abbreviation? Or is this particular spelling required by Superlu_DIST?